duktape.mk

```diff
@@ -10,7 +10,9 @@ duktape-setup: setup
 	wget -q -nc -P $(BUILD_SOURCE) https://github.com/svaarala/duktape/releases/download/v$(DUKTAPE_VERSION)/duktape-$(DUKTAPE_VERSION).tar.xz
 	$(call EXTRACT_TAR,duktape-$(DUKTAPE_VERSION).tar.xz,duktape-$(DUKTAPE_VERSION),duktape)
 	$(SED) -i 's/gcc/cc/g' $(BUILD_WORK)/duktape/Makefile.cmdline
+	$(SED) -i 's/gcc/cc/g' $(BUILD_WORK)/duktape/Makefile.sharedlibrary
 	$(SED) -i 's|$$(CCLIBS)|$$(CCLIBS) $(LDFLAGS) $(CFLAGS)|g' $(BUILD_WORK)/duktape/Makefile.cmdline
+	$(SED) -i 's|$$(CC)|$$(CC) $(LDFLAGS) $(CFLAGS)|g' $(BUILD_WORK)/duktape/Makefile.sharedlibrary
```

Comment:
…how is the value of $(CC) in the Makefile already? This won’t do anything on Linux.